NewSpring/corinth
# chore: Update build.gradle to support Android 10 (API 29)

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates the Android app to support Android 10 (Android SDK 29).

### How do I test this PR?
Run the app in Android Studio (or by running `yarn android` in terminal) and make sure that all the things still work.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
